src/features/pagination/js/pagination.js

```diff
@@ -368,8 +410,6 @@
           $scope.$on('$destroy', dataChangeDereg);
 
           var setShowing = function () {
```

Comment:
That was in the original code; I could not find a usage or reassignment of it, but I didn't want to touch it in case I was wrong.